src/Sass/Components/FlashCards.scss

```diff
@@ -0,0 +1,37 @@
+.card {
+  width: 80vw;
+  padding: 5rem;
+  display: flex;
+  align-items: center;
+  font-size: 1.75rem;
+  font-weight: 300;
+
+
+  &__header {
```

Comment:
Why the __?

Reply:
Block Element Modifier styling for naming in html example: 
`
<div clasName = 'card'> 
      <div className='card__item'> 
      </div>
       <div className='card__item-varient'> 
      </div>
</div>
`